graphbolt/src/csc_sampling_graph.cc

```diff
@@ -517,10 +517,12 @@ torch::Tensor Pick<SamplerType::LABOR>(
     return picked_neighbors;
   } else if (replace) {
     return LaborPick<false, true>(
-        offset, num_neighbors, fanout, options, probs_or_mask, args);
+        offset, num_neighbors, fanout, options,
+        /* probs_or_mask= */ torch::nullopt, args);
```

Comment:
Why do we need to disable weighted sampling with replacement? It should be functional already and working.